cpp/ql/test/library-tests/ir/ir/PrintConfig.qll

```diff
@@ -1,10 +1,17 @@
 private import cpp
 
+/**
+ * Holds if an AST or IR with the specified location should be printed in the test output.
+ *
+ * This predicate excludes locations in standard headers.
+ */
+predicate shouldDump(Location loc) {
```

Comment:
Should we name this `shouldDumpLocation` to be more consistent with `shouldDumpFunction`?

Reply:
makes sense